code/modules/mob/logout.dm

```diff
@@ -1,7 +1,10 @@
 /mob/Logout()
 	SSnano.user_logout(src) // this is used to clean up (remove) this user's Nano UIs
 	if(interactee) 
-		unset_interaction()
+		unset_interaction()	
+	if(typing)
+		remove_emote_overlay(client, typing, viewers())
```

Comment:
```suggestion
		remove_emote_overlay(null, typing, viewers())
```
The client won't be present. Null will work since there's no early return in the proc.

```
/mob/proc/remove_emote_overlay(client/C, image/emote_overlay, list/viewers)
	if(C)
		C.images -= emote_overlay
	for(var/mob/M in viewers)
		if(M.client)
			M.client.images -= emote_overlay
	qdel(emote_overlay)
```

Reply:
Done! :)